ROADMAP.md

```diff
@@ -0,0 +1,66 @@
+# AICSPyLibCZI Roadmap
+
+## For 3.* Releases 
+The `aicspylibczi` roadmap captures current development priorities within the project
+and should serve as a guide for core developers, to encourage contribution for new
+contributors, to provide insight to external developers who are interested in using
+`aicspylibczi` in their work, and to provide deeper insights to developers using 
+`aicsimageio` which uses this library to read ZEISSRAW format CZI files.
+
+The *mission* of `aicspylibczi` is to provide a robust method of access to Zeiss's 
+CZI microscopy files. Our approach is to use Zeiss's libCZI and leverage that to 
+create a C++ tool that extracts the desired image data and metadata in a structure
+that performs well with respect to speed and memory usage and enables mapping into
+a python numpy.ndarray. The bindings to create a python module are done using 
+pybind11 and are then wrapped in python class providing a more pythonic interface
+into the module that the direct bindings.
+
+Our goal with the 3.* API changes is:
+ 
+ * Unify the function/property naming conventions
+ * Modify the library to support Samples, the A channel for RGB/BGR images
+ * Enable reading of attachment data
+ * Provide sufficient information to enable `aicspylibczi` to load a mosaic image as a dask.ndarray
+ * Enable automated benchmark scripts
+
+## Unify the function/property naming conventions
+Before 3.* it was pointed out the interface was a bit inconsistent. We attempt to address
+that here by using only bounding boxes and providing matched methods specific for non-mosaic
+images and mosaic images.
+
+## Modify the library to support Samples, the A channel for RGB/BGR images
```

Comment:
instead of `the A channel` it should probably say `the A dimension`... the A channel makes it sound like alpha.